Makefile

```diff
@@ -195,6 +195,11 @@ publishing-bot:
 	$(MAKE) apply WHAT=projects/publishing-bot/storage-class.yaml
 .PHONY: publishing-bot
 
+image-registry-publishing-bot:
```

Comment:
You also need to automate the creation of the namespace where you want to run the bot, otherwise it will land on whatever ns is in the context when we run make.

Reply:
@kargakis ptal at cluster/ci/config/roles.yaml, is it a correct way to create the namespace?

Reply:
Looks good, the last thing I see missing is the secret creation. I think it's fine to create it in this target using an ENV you expect us to have set and we need to figure out a holistic way for getting all the secrets in the cluster.